pkgs/top-level/all-packages.nix

```diff
@@ -42344,4 +42344,6 @@ with pkgs;
   ssl-proxy = callPackage ../tools/networking/ssl-proxy { };
 
   code-maat = callPackage ../development/tools/code-maat {};
+
+  tradingview = callPackage ../development/tools/tradingview {  };
```

Comment:
non-blocking: you could consider adding this to `pkgs/by-name instead`

see: https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md